Update service_definitions.md - add v3 details

crunchypistachiobutter opened the pull request:
add v3.0 schema details and example

<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
add details for schema v3.0

### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [x] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

drichards-87 commented:
Created DOCS-7867 for editorial review.

domalessi commented on content/en/service_catalog/service_definitions.md:
```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
 ### Metadata Schema v3.0 (beta) 
 The Entity Definition Schema is a structure that contains basic information about an entity. See the [full schema on GitHub][1].
 
+#### Example 
+
+##### YAML for `kind:application`
```
Are you intending for there to be additional examples added down the road? Otherwise, I would suggest removing the "YAML for `kind:application`" header and combining it with the "Example" header. So something like "Example YAML for `kind:application`"

domalessi commented on content/en/service_catalog/service_definitions.md:
```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
+      - fp2
+{{< /code-block >}}
+
+#### Common Questions
+##### How do I specify common components (e.g., orders-postgres) that are part of multiple applications
```
Since there's only one question here, I would recommend having a single header that says something like "Specify common components that are part of multiple applications"

domalessi commented on content/en/service_catalog/service_definitions.md:
```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
+
+#### Common Questions
+##### How do I specify common components (e.g., orders-postgres) that are part of multiple applications
+For example, if orders-postgres is a part of the overall postgres fleet managed by a database admin or infrastructure team, but also a critical part of the shopping-cart application. In this case, you will specify two application YAMLs. 
```
```suggestion
For example, if the datastore `orders-postgres` is a component of both a postgres fleet and a web application, specify two YAMLs:
```

domalessi commented on content/en/service_catalog/service_definitions.md:
```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
+##### How do I specify common components (e.g., orders-postgres) that are part of multiple applications
+For example, if orders-postgres is a part of the overall postgres fleet managed by a database admin or infrastructure team, but also a critical part of the shopping-cart application. In this case, you will specify two application YAMLs. 
+
+One for the postgres fleet (managed-postgres):
```
```suggestion
For the postgres fleet (`managed-postgres`):
```

domalessi commented on content/en/service_catalog/service_definitions.md:
```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
+  owner: db-team
+{{< /code-block >}}
+
+A separate one for the shopping-cart application:
```
```suggestion
One for the web application (`shopping-cart`):
```

domalessi reviewed (commented):
Left a few points of feedback. Let me know if you have any questions!

crunchypistachiobutter commented:
Have reviewed and addressed the comments! Thank you!

On Fri, May 3, 2024 at 6:39 PM domalessi ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Left a few points of feedback. Let me know if you have any questions!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DataDog/documentation/pull/22980#pullrequestreview-2039083561>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVIISIW4AJG4LNUFNLXRYLLZAQG2PAVCNFSM6AAAAABHELZYTOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDAMZZGA4DGNJWGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

domalessi reviewed (approved):
Thanks for the changes! Looks good to me.

crunchypistachiobutter commented:
Ready to merge!

On Mon, May 6, 2024 at 4:49 PM domalessi ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Thanks for the changes! Looks good to me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DataDog/documentation/pull/22980#pullrequestreview-2041652381>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVIISIWMBSJL57BXW5D3FQ3ZA7UFDAVCNFSM6AAAAABHELZYTOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDANBRGY2TEMZYGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
